x-pack/platform/plugins/shared/alerting/server/application/maintenance_window/methods/create/create_maintenance_window.test.ts

```diff
@@ -283,53 +283,6 @@ describe('MaintenanceWindowClient - create', () => {
     `);
   });
 
-  it('should throw if trying to create a MW with a scoped query with other than 1 category ID', async () => {
```

Comment:
This validation was removed so that the new POST maintenance window API could be created.
The `categoryIDs` field will be completely removed in the future.

https://github.com/elastic/kibana/issues/197530